tests/D2L.CodeStyle.Analyzers.Test/Immutability/UnsafeStaticsAnalyzerTests.cs

```diff
@@ -83,13 +83,15 @@ public sealed class Foo {
         }
     }";
             var diag1 = CreateDiagnosticResult( 20, 44, "Default", MutabilityInspectionResult.Mutable(
-                mutableMemberPath: "Default.uhoh",
+				null,

```

Comment:
Symbol not used by analyzer yet

Reply:
Do you want tabs? I'm seeing a mix of tabs and spaces throughout the project

Reply:
D2L standard is tabs, feel free to reformat files as you see them.